README.md

```diff
@@ -103,6 +103,7 @@ To integrate Velero with GCP, create a Velero-specific [Service Account][15]:
         compute.snapshots.useReadOnly
         compute.snapshots.delete
         compute.zones.get
+        storage.objects.list
```

Comment:
@pantelis-karamolegkos 
Thanks for your PR.
Running the command `gsutil iam ch serviceAccount:$SERVICE_ACCOUNT_EMAIL:objectAdmin gs://${BUCKET}` from the `README.md` grants the service account `objectAdmin` privileges to the bucket used to store Velero's backups.

Can you please confirm if this is not sufficient?

Reply:
@pantelis-karamolegkos wanted to double check on this. The current instructions have been battled tested and work. Were you able to run the cmd above as per what's in the instructions and have it work without errors?

Adding `storage.objects.list` might make that error go away but not provide sufficient permission as the `objectAdmin` does.